### ISSUE TITLE
[Bug] `model` variable in `GeoEntityRenderer` is used for animations instead of the return value of the `getGeoModel` method

### ISSUE THREAD

**Issue by thomasglasser** [bernie-g/geckolib#575]
The entity animations are handled by the model variable instead of the method, so overriding it doesn't change animations used.

`GeoEntityRenderer` lines 218 & 219:
```java
			this.model.addAdditionalStateData(animatable, instanceId, animationState::setData);
			this.model.handleAnimations(animatable, instanceId, animationState);
```

**Comment by Tslat** [comment]
What version of geckolib is this in?

**Comment by thomasglasser** [comment]
4.4.4, mc 1.20.4

**Comment by Tslat** [comment]
This was fixed in 4.5

**Comment by Tslat** [comment]
You should be moving on from 1.20.4 lol
